rust-runtime/aws-smithy-http-server/src/runtime_error.rs

```diff
@@ -25,7 +25,8 @@ use crate::protocols::Protocol;
 
 #[derive(Debug)]
 pub enum RuntimeErrorKind {
-    // UnknownOperation,
+    /// The requested operation does not exist.
+    UnknownOperation,
```

Comment:
Is there a unit test in the router checking that this gets hit when the operation does not exist?

Reply:
Nope, there isn't. I'll add one.